msteams-platform/overview.md

```diff
@@ -203,6 +203,13 @@ Looking for app development inspiration? Browse our list of real-world scenarios
    :::column-end:::
 :::row-end:::
 
+## Integrating Teams communication in an external app
+This page focuses on embedding your own experiences into Microsoft Teams. If you'd like to reverse this model and integrate Teams or other communication capabilities into your own app experience, check out [Azure Communication Services](https://docs.microsoft.com/azure/communication-services/overview).
+
+Azure Communication Services are cloud-based services with REST APIs and client library SDKs to help you integrate communication into your own custom applications. A [UI library](https://azure.github.io/communication-ui-library/) for desktop and mobile web browsers can help you quickly embed generic or Teams-styled components for chat and calling.
```

Comment:
You can embed generic or Teams-styled components for calling and chatting with the help of [UI library](https://azure.github.io/communication-ui-library/).